### ISSUE TITLE
NoneType error triggered by sphinx theme in v1.6.1

### ISSUE THREAD

**Issue by jputrino** [sphinx-doc/sphinx#3773]
Subject: NoneType error breaks builds in v1.6.1

### Problem
When I try to build html documentation, a NoneType error is generated that breaks the build.

Note: I ran the command in a docker container based on Ubuntu xenial and on Mac OS Sierra, with the same results. It's also breaking the build in travis here: 
https://travis-ci.org/F5Networks/f5-openstack-heat-plugins/builds/234084450

#### Procedure to reproduce the problem
```
# after upgrade to 1.6.1
./scripts/docker-docs.sh sphinx-build -b html -aE docs/ docs/_build
+ : docker-registry.pdbld.f5net.com/jputrino/containthedocs:issue8-38346
++ id -u
+ exec docker run --rm -it -v /Users/putrino/velcro/f5-ci-docs:/Users/putrino/velcro/f5-ci-docs --workdir /Users/putrino/velcro/f5-ci-docs /jputrino/containthedocs:issue8-38346 make -C docs/ html
Starting with UID : 25231192
make: Entering directory '/Users/putrino/velcro/f5-ci-docs/docs'
sphinx-build -b html -d _build/doctrees  -aE . _build/html
Running Sphinx v1.6.1
loading translations [en]... done
building [mo]: all of 0 po files
building [html]: all source files
updating environment: 28 added, 0 changed, 0 removed
reading sources... [100%] tutorials/how-to-send-stats-to-Splunk
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [  3%] acknowledgements
Exception occurred:
  File "/usr/local/lib/python2.7/dist-packages/sphinx/themes/classic/../basic/layout.html", line 108, in template
    {%- for css in css_files %}
TypeError: 'NoneType' object is not iterable
The full traceback has been saved in /tmp/sphinx-err-DxyVys.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
Makefile:54: recipe for target 'html' failed
make: *** [html] Error 1
make: Leaving directory '/Users/putrino/velcro/f5-ci-docs/docs'
```

#### Error logs / results
```
# Sphinx version: 1.6.1
# Python version: 2.7.12 (CPython)
# Docutils version: 0.13.1 release
# Jinja2 version: 2.9.6
# Last messages:
#
#   looking for now-outdated files...
#   none found
#   pickling environment...
#   done
#   checking consistency...
#   done
#   preparing documents...
#   done
#   writing output... [  3%] acknowledgements
# Loaded extensions:
#   sphinx.ext.coverage (1.6.1) from /usr/local/lib/python2.7/dist-packages/sphinx/ext/coverage.pyc
#   sphinx.ext.todo (1.6.1) from /usr/local/lib/python2.7/dist-packages/sphinx/ext/todo.pyc
#   sphinx.ext.intersphinx (1.6.1) from /usr/local/lib/python2.7/dist-packages/sphinx/ext/intersphinx.pyc
#   sphinx.ext.doctest (1.6.1) from /usr/local/lib/python2.7/dist-packages/sphinx/ext/doctest.pyc
#   alabaster (0.7.10) from /usr/local/lib/python2.7/dist-packages/alabaster/__init__.pyc
#   sphinx.ext.ifconfig (1.6.1) from /usr/local/lib/python2.7/dist-packages/sphinx/ext/ifconfig.pyc
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/sphinx/cmdline.py", line 306, in main
    app.build(opts.force_all, filenames)
  File "/usr/local/lib/python2.7/dist-packages/sphinx/application.py", line 332, in build
    self.builder.build_all()
  File "/usr/local/lib/python2.7/dist-packages/sphinx/builders/__init__.py", line 285, in build_all
    self.build(None, summary='all source files', method='all')
  File "/usr/local/lib/python2.7/dist-packages/sphinx/builders/__init__.py", line 394, in build
    self.write(docnames, list(updated_docnames), method)
  File "/usr/local/lib/python2.7/dist-packages/sphinx/builders/__init__.py", line 431, in write
    self._write_serial(sorted(docnames))
  File "/usr/local/lib/python2.7/dist-packages/sphinx/builders/__init__.py", line 440, in _write_serial
    self.write_doc(docname, doctree)
  File "/usr/local/lib/python2.7/dist-packages/sphinx/builders/html.py", line 556, in write_doc
    self.handle_page(docname, ctx, event_arg=doctree)
  File "/usr/local/lib/python2.7/dist-packages/sphinx/builders/html.py", line 940, in handle_page
    output = self.templates.render(templatename, ctx)
  File "/usr/local/lib/python2.7/dist-packages/sphinx/jinja2glue.py", line 176, in render
    return self.environment.get_template(template).render(context)
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/sphinx/themes/basic/page.html", line 10, in top-level template code
    {%- extends "layout.html" %}
  File "/usr/local/lib/python2.7/dist-packages/f5_sphinx_theme/layout.html", line 3, in top-level template code
    {% set css_files = css_files + ["_static/css/f5.css", "_static/css/custom.css"] %}
  File "/usr/local/lib/python2.7/dist-packages/sphinx/themes/classic/../classic/layout.html", line 13, in top-level template code
    {% set script_files = script_files + ['_static/sidebar.js'] %}
  File "/usr/local/lib/python2.7/dist-packages/sphinx/themes/classic/../basic/layout.html", line 128, in top-level template code
    {%- block csss %}
  File "/usr/local/lib/python2.7/dist-packages/sphinx/themes/classic/../basic/layout.html", line 129, in block "csss"
    {{ css() }}
  File "/usr/local/lib/python2.7/dist-packages/jinja2/sandbox.py", line 427, in call
    return __context.call(__obj, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/jinja2/runtime.py", line 553, in _invoke
    rv = self._func(*arguments)
  File "/usr/local/lib/python2.7/dist-packages/sphinx/themes/classic/../basic/layout.html", line 108, in template
    {%- for css in css_files %}
TypeError: 'NoneType' object is not iterable
```
- <public link of unexpected result if you have>

#### Expected results
successful build

### Reproducible project / your project
- the project referenced above is private, but you can use [f5-openstack-lbaasv2-driver](https://github.com/f5networks/f5-openstack-lbaasv2-driver) (it uses the same theme and tools).

### Environment info
- OS: Mac OS Sierra 10.12.5 / `ubuntu:xenial`-based docker image
- Python version: 2.7.12
- Sphinx version: 1.6.1
- [f5-sphinx-theme](https://github.com/f5devcentral/f5-sphinx-theme) @ master

**Comment by tk0miya** [comment]
Sorry for inconvenience.
That was already fixed at #3754.

**Comment by jputrino** [comment]
thanks @tk0miya!
